include/drivers/i2c/slave_eeprom.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2017 BayLibre, SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef I2C_SLAVE_EEPROM_H
+#define I2C_SLAVE_EEPROM_H
+
+typedef int (*eeprom_slave_program_t)(struct device *dev, u8_t *data,
+				      unsigned int length);
+typedef int (*eeprom_slave_read_t)(struct device *dev, u8_t *data,
+				   unsigned int offset);
+
+struct eeprom_slave_api {
```

Comment:
Is it necessary to be inconsistent with

slave_eeprom
eeprom_slave

?

Reply:
You are right, it was in my todo list